js/details.js

```diff
@@ -170,11 +178,7 @@
       return;
     }
 
-    window.removeEventListener('AutoTheme:palette', Details.onPalette);
-    Navigation.pop().then(() => {
-      AutoTheme.clean();
-      AutoTheme.showPalette(Details.autotheme);
-    });
+    Navigation.pop();
```

Comment:
so now we always have the same code to handle the "action". Maybe (not for this patch) we could try to factorize this. I don't have very good ideas to do this, only some ideas that do not please me a lot around inheritance. Maybe creating a component inheriting from gaia-header ? Or doing a BasePanel whose our Panel would inherit from in some way?
Yeah, I said it: nothing really please me :)